Makefile

```diff
@@ -5,7 +5,7 @@ override SPACY_EXTRAS = spacy-lookups-data==1.0.2 jieba spacy-pkuseg==0.0.28 sud
 endif
 
 ifndef PYVER
-override PYVER = 3.6
+override PYVER = 3.8
```

Comment:
```suggestion
override PYVER = 3.6
```

Reply:
You should be able to just commit this suggestion (I don't have write access to your branch)